src/vs/workbench/contrib/positronOutputWebview/browser/notebookOutputWebviewServiceImpl.ts

```diff
@@ -165,15 +176,12 @@ export class PositronNotebookOutputWebviewService implements IPositronNotebookOu
 	}
 
 	/**
-	 * Gets renderer data for a given MIME type. This is used to inject only the
-	 * needed renderers into the webview.
+	 * Gets renderer data. This is used to inject renderer contexts into the webview.
 	 *
-	 * @param mimeTypes The MIME types to get renderers for
-	 * @returns An array of renderers that can render the given MIME type
+	 * @returns An array of renderer metadata.
 	 */
-	private getRendererData(mimeTypes: string[]): RendererMetadata[] {
+	private getRendererData(): RendererMetadata[] {
 		return this._notebookService.getRenderers()
-			.filter(renderer => mimeTypes.some(mimeType => renderer.mimeTypes.includes(mimeType)))
```

Comment:
Is this so the webview is always ready to render any kind of message internally, since we don't know what will be needed for interactive widgets?

Reply:
Yep. It also shouldn't have any performance overhead since renderer modules aren't actually imported or activated until the renderer is used. The back layer webview does the same.